wallet/src/repository/wallet/mod.rs

```diff
@@ -514,6 +517,7 @@ where
             batch.put(keys::account_balance(account), new_balance)?;
             batch.put(keys::account_utxo_set(account), db_utxo_set)?;
             batch.put(keys::transaction_next_id(account), txn_next_id)?;
+            batch.put(keys::transactions_index(txn_hash), ())?;
```

Comment:
Could you add a comment here?

```
// Mark transaction as indexed so we don't accidentally index it twice
```

Reply:
Now I'm associating the transaction id to its hash, so I believe the comment is no longer needed